dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -926,7 +936,8 @@ def prepare_provider_packages(
         include_removed=include_removed_providers,
         include_not_ready=include_not_ready_providers,
     )
-    if not skip_tag_check:
+    package_version = create_package_version(version_suffix_for_pypi, version_suffix_for_local)
+    if not skip_tag_check and not is_local_package_version(package_version):
         run_command(["git", "remote", "rm", "apache-https-for-providers"], check=False, stderr=DEVNULL)
         make_sure_remote_apache_exists_and_fetch(github_repository=github_repository)
```

Comment:
I'm not totally convinced we want to ignore the github_repository flag if it is a local build any more. When it was just a local provider build, it seemed okay. Now I have concerns and I'm not sure if I need to investigate this. Any thoughts, @potiuk?

Reply:
Yes. We can skip it. This is only useful to skip providers automatically when they are already released.

Reply:
Thank you. That was what my initial analysis showed, but I'm new to this....